### PR TITLE
LPS-69325 Rewrites Separator field getValeu due this field has no input element.

### PR DESCRIPTION
**Pull request by fernandosouza**

### CODE REVIEW

**Comment by brunobasto** [comment]
Just started reviewing :) <br><br>:octocat: *Sent from [GH](http://nodegh.io).*

**Comment by brunobasto** [comment]
ci:reopen

**Comment by marcellustavares** [comment]
@brunobasto @fernandosouza tests are missing.

**Comment by brunobasto** [comment]
@marcellustavares For this code, better to add Poshi tests.

**Comment by marcellustavares** [comment]
cool, thx @brunobasto!

File: modules/apps/forms-and-workflow/dynamic-data-mapping/dynamic-data-mapping-web/src/main/resources/META-INF/resources/js/ddm_form.js [hunk 1]
```diff
@@ -2744,6 +2744,20 @@ AUI.add(
 
 		FieldTypes.select = SelectField;
 
+		var FieldSeparator = A.Component.create(
```
Comment (fernandosouza) [inline modules/apps/forms-and-workflow/dynamic-data-mapping/dynamic-data-mapping-web/src/main/resources/META-INF/resources/js/ddm_form.js]: Sure, @brunobasto. Done

**Comment by brunobasto** [comment]
Just started reviewing :) <br><br>:octocat: *Sent from [GH](http://nodegh.io).*

**Comment by brunobasto** [comment]
Pull request submitted to https://github.com/vicnate5/liferay-portal/pull/642. [See changes here.](https://github.com/brunobasto/liferay-portal/compare/2b0a719~0...2b0a719) <br><br>:octocat: *Sent from [GH](http://nodegh.io).*

**Pull request closed without merging**